src/components/Icons/utils.ts

```diff
@@ -12,3 +13,11 @@ export function getFill({ gradient, color, id }: getFillParams) {
   }
   return color;
 }
+
+export function getViewBox(size: number = sizes.smaller) {
+  return `0 0 ${size} ${size}`;
+}
+
+export function getSize(size: number = sizes.smaller) {
+  return size;
+}
```

Comment:
⛏️ I think the best way to define these properties is limit it to our size tokens instead of allow any size.
Using our size tokens will guarantee that, wherever used, will be consistent with our design system sizes.

Reply:
I agree, but what is the best way to do this?
the idea was to use the token in the function call

Reply:
You can create a map object that could act like a glue between a prop value and the value itself.
```suggestion
type sizeTokens = keyof typeof sizes;
 
export function getViewBox(token: sizeTokens = 'smaller') {
  return `0 0 ${sizes[token]} ${sizes[token]}`;
}

export function getSize(token: sizeTokens = 'smaller') {
  return sizes[token];
}
```

Probably this can lead to a new refactor in all icons to change size prop to a token key. If this is your case, you can do it in another PR to simplify this and isolate contexts (this PR includes new icons and new PR refactor all icons).
Maintain the retrocompatibility with the current API will be a nice plus 🌱

Reply:
comment resolved in https://github.com/magnetis/astro-native/pull/538/commits/4efc4f54ee6817280e4709ae60ec68f90ffcd8fe